chia/full_node/mempool_manager.py

```diff
@@ -324,23 +277,25 @@ async def pre_validate_spendbundle(
 
         self._worker_queue_size += 1
         try:
-            err, cached_result_bytes, new_cache_entries, duration = await asyncio.get_running_loop().run_in_executor(
+            sbc, new_cache_entries, duration = await asyncio.get_running_loop().run_in_executor(
                 self.pool,
                 validate_clvm_and_signature,
-                new_spend_bytes,
+                new_spend,
```

Comment:
This function still takes `new_spend_bytes` as an argument, and if it's `None` it still serializes `new_spend` into it.
I think this can (and should) be removed now. We would need to be sure that no caller ever passes in *just* the serialized form of the spend, but that `new_spend` always is valid.